src/frontend/hooks/useDraftObservation.tsx

```diff
@@ -141,13 +151,12 @@ export default (): UseDraftObservation => {
   const newDraft = useCallback(
     (
       id?: string,
-      value?: ObservationValue | null = { tags: {} },
+      value: ObservationValue | (null | undefined) = { tags: {} },
```

Comment:
nit but could keep the `value?` here

Reply:
Idk, why but this is the error I am getting if I left `value?`
![image](https://user-images.githubusercontent.com/67773827/142958439-76cff772-16ae-463d-a99e-688935567df7.png)

Reply:
Oh yeah, it looks like because we are initializing a value, there is no point of making it optional

Reply:
oh yeah of course 🤦‍♂️ nvm then what you have there makes sense